02-microarray/ortholog-mapping_microarray_01_ensembl.Rmd

```diff
@@ -122,13 +122,13 @@ This is handy to do because if we want to switch the dataset (see next section f
 
 ```{r}
 # Define the file path to the data directory
-data_dir <- file.path("data", "GSE13490") # Replace with accession number which will be the name of the folder the files will be in
+data_dir <- file.path("data", "GSE71270") # Replace with accession number which will be the name of the folder the files will be in
```

Comment:
Are these accession number changes correct?

Reply:
Yes. And that's the scary part. I accidentally changed these for #298 and this didn't throw an error since this analysis doesn't have an particular step that was affected by it being a different species.